gio/src/application.rs

```diff
@@ -12,12 +12,19 @@ use std::mem::transmute;
 
 pub trait ApplicationExtManual {
     #[doc(alias = "g_application_run")]
-    fn run(&self, argv: &[String]) -> i32;
+    fn run(&self) -> i32;
+    #[doc(alias = "g_application_run")]
+    fn run_with_args<S: AsRef<str>>(&self, args: &[S]) -> i32;
```

Comment:
I'm not a big fan of the naming. What do you think about `run_with_environment_args`?

Reply:
Instead of plain `run()`? That's a lot of additional typing for the normal case :)

Reply:
I made my comment on `run_with_args`. :stuck_out_tongue:

Reply:
It's not using the environment args, but the ones you pass in.

Reply:
Oh indeed, just realized that it was using argc and argv, not the env. My bad!